### PR TITLE
mod add new riverbank battle map

### PR DESCRIPTION
**Pull request by namidaka**

### CODE REVIEW

File: src/Module.Client/GUI/MapSpriteData.xml [hunk 1]
```diff
@@ -1103,6 +1104,15 @@
       <SheetY>0</SheetY>
       <CategoryName>ui_mploading</CategoryName>
     </SpritePart>
+    <SpritePart>
```
Comment (verdie-g) [inline src/Module.Client/GUI/MapSpriteData.xml:1107]: It also needs a GenericSprite at the end of the file

File: src/Module.Server/ModuleData/multiplayer_strings.xml [hunk 2]
```diff
@@ -112,6 +112,7 @@
   <string id="str_multiplayer_scene_name.crpg_battle_weirwater" text="{=NuReq6rk}Weirwater" />
   <string id="str_multiplayer_scene_name.crpg_battle_whitebridge" text="{=jP0ySZzV}White Bridge" />
   <string id="str_multiplayer_scene_name.crpg_battle_zemouri" text="{=LLWi2bYx}Zemouri" />
+  <string id="str_multiplayer_scene_name.crpg_battle_riverbank" text="Riverbank" />
```
Comment (verdie-g) [inline src/Module.Server/ModuleData/multiplayer_strings.xml:115]: NIT: order the maps alphabetically

Comment (verdie-g) [reply src/Module.Server/ModuleData/multiplayer_strings.xml:115]: The map needs also to be translated to Chinese and Russian:
- 河岸
- Речной берег

**Review by verdie-g — CHANGES_REQUESTED** [review]

File: src/Module.Client/GUI/MapSpriteData.xml (hunk 1 above)
Comment (namidaka) [reply src/Module.Client/GUI/MapSpriteData.xml:1107]: done

**Review by verdie-g — APPROVED** [review]

**Pull request merged**